### ISSUE TITLE
[ACCESSIBILITY] Unable to click "Vote" on Quick polls using TalkBack

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1494]
**Describe the accessibility issue**
Unable to click "Vote" on Quick polls using TalkBack.

**Steps to reproduce the behavior:**
1. Go to 'Quick Polls.'
2. Swipe through options, as they are read, listen for Vote. 
3. When user lands on 'Vote.' double tap to select

**Expected behavior**
Upon TalkBack users' double tap on 'Vote' some status message should be announced letting user know their vote was accepted. Then, they should be redirected to the Poll page with their vote showing in the count of votes for that Poll.  

**Actual Behavior**
Upon TalkBack users double tap on 'Vote' some status message should be announced letting user know their vote was accepted. I hear nothing. I am not taken back to Poll page with my vote shown in the count of votes for Poll. 

**Screenshots**
![Screenshot_20220323-105610](https://user-images.githubusercontent.com/82892873/160190342-6aea8432-1e3a-437e-8d06-f56ebf19b39c.png)

**Smartphone (please complete the following information):**
iPhone XR- iOS 14.6, Pixel 4a- Android 12

**Additional context**

**Comment by acovher2** [comment]
Verified as resolved in 3.3.8. Closing.
